sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/RewriteDistinctAggregatesSuite.scala

```diff
@@ -63,7 +63,7 @@ class RewriteDistinctAggregatesSuite extends PlanTest {
     val input = testRelation
       .groupBy('a, 'd)(
         countDistinct('e, 'c).as('agg1),
-        CollectSet('b).toAggregateExpression().as('agg2))
+        DummpAgg('b).toAggregateExpression().as('agg2))
```

Comment:
If we remove the logic from RewriteDistinctAggregates, we need to make sure that non-partial aggregates do not exist anymore. Lets do this in a follow-up.